src/main/cpp/RLBotInterface/BallPrediction/inc/vec.h

```diff
@@ -0,0 +1,215 @@
+#pragma once
+
+#include <math.h>
+#include <iostream>
+#include <initializer_list>
+
+template < int n > 
+class vec {
```

Comment:
if we are treating this as a n dimensional mathematical vector and making this different than the vector3 implementation can we add comments and naming to make that more obvious?

Reply:
I only ever use vec < 3 > ("vec3") for RLBot